web_graph_improved/README.md

```diff
@@ -0,0 +1,7 @@
+Charts, Improved.
+=================
+
+Various improvements to the graph view:
+
+ * Bar: can plot multiple variables (called measures in the odoo client)
+   when there is only one row group
```

Comment:
Saw the description in the PR initial comment. The description here should be improved a bit, IMO.